vnext/Microsoft.ReactNative/Views/Image/ImageViewManager.cpp

```diff
@@ -138,6 +138,13 @@ bool ImageViewManager::UpdateProperty(
   } else if (propertyName == "tintColor" && IsValidColorValue(propertyValue)) {
     auto reactImage{grid.as<ReactImage>()};
     reactImage->TintColor(ColorFrom(propertyValue));
+  } else if (propertyName == "isPlaying") {
+    auto reactImage{grid.as<ReactImage>()};
+    if (propertyValue.Type() == winrt::Microsoft::ReactNative::JSValueType::Boolean) {
+      reactImage->IsPlaying(propertyValue.AsBoolean());
+    } else if (propertyValue.Type() == winrt::Microsoft::ReactNative::JSValueType::Null) {
+      reactImage->IsPlaying(true);
```

Comment:
I noticed other props do not accept null updates to set the native control back to default behavior. E.g. tintColor will noop unless a color is given.

Are the pre-existing props bugged when clearing them out?

Reply:
I'm sure there are quite a few bugs when clearing out props :)